explorer/events.go

```diff
@@ -220,7 +220,7 @@ func AppliedEvents(cs consensus.State, b types.Block, cu ChainUpdate) []Event {
 		var e EventV2Transaction
 		for _, a := range txn.Attestations {
 			var ha chain.HostAnnouncement
-			if ha.FromAttestation(a) {
+			if ha.FromArbitraryData(a.Value) {
```

Comment:
Need to use V2HostAnnouncement.FromAttestation and a.PublicKey here I think.  Will push a fix.  Rest looks good to me

Reply:
Ok. Feel free to take over this PR. I am kinda surprised no test failed though.

Reply:
Attestations/host announcements aren't supported in master yet.  A host announcement test is in the v2 branch which I confirmed breaks with this change.  I kinda want to change the base branch of this PR to v2 since I'm almost done with v2 changes anyways.

Reply:
AppliedEvents is called on the raw block.  If a v2 host announcement is in a block, we won't store it in the DB, but AppliedEvents will panic.